src/components/Map.vue

```diff
@@ -61,10 +61,19 @@ export default {
       }
       const template = {
       // autocasts as new PopupTemplate()
-        title: '{BLOCK}', // Show attribute value
+        title: '{ID}', // Show attribute value
         content: getModalContent(),
         actions,
       };
+      const blockLabel = {
```

Comment:
So I don't know if we want to actually show the ID on the map (I think that's this line?) 
We should keep this functionality toggleable thought for sure.

Reply:
cc @liammoyn @florisdobber

Reply:
It's a lot easier for people to identify blocks if the ID is shown, but if users zoom out too much the map might get cluttered. So making it toggle-able definitely seems like the way to go, and we can just have a button right above the map that toggles this.

Reply:
Awesome, let's stick with having them on the map but keeping it toggleable.

@willmt80 how long do you think it will take?

Reply:
I will look into it, shouldn't take all that long to change

Reply:
So the toggle works but it's not the cleanest solution